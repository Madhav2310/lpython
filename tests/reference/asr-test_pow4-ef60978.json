{
    "basename": "asr-test_pow4-ef60978",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pow4.py",
    "infile_hash": "a7047d2e1e56bc3d001973391a5ce253ece0b92c553d488a3c255dee",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_pow4-ef60978.stderr",
    "stderr_hash": "10a6c138d7353eb26a11f342c4ac31d305d28d47f7f65a2f2ccb4158",
    "returncode": 2
}