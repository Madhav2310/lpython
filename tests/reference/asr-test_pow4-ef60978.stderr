semantic error: Only Name, Subscript, and Call supported for now in annotation of annotated assignment.
 --> tests/errors/test_pow4.py:4:5
  |
4 |     a: 2
  |     ^^^^ 
